src/app/core/services/basic-rundown-state.service.ts

```diff
@@ -7,6 +7,7 @@ import { BasicRundown } from '../models/basic-rundown'
 import { BasicRundownService } from '../abstractions/basic-rundown.service'
 import { RundownEvent } from '../models/rundown-event'
 import { BasicRundownEntityService } from './models/basic-rundown-entity.service'
+import { Logger } from '../../../../../../mediatech-logger'
```

Comment:
Is this right?
`Logger` is declared in `core/abstractions/logger.service`.

You have quite a few of these imports. (And some imports that uses `core/abstractions/logger.service`, so something isn't right)

Reply:
That is incorrect and is fixed.